kyaml/kio/filters/container_test.go

```diff
@@ -773,6 +773,102 @@ metadata:
 `, b.String())
 }
 
+func TestFilter_Filter_scope_nested_resource(t *testing.T) {
```

Comment:
Maybe you want to combine the `TestFilter_Filter_scope_nested_resource` and `TestFilter_Filter_scopeDir` since they only diff for the `config.kubernetes.io/path` in the config resource.

Reply:
I rather have it separate, generally speaking duplication in tests is not a big deal (compared to having conditionals to combine multiple test cases into one).

Reply:
SG